ocaml/xapi-cli-protocol/tar_helpers.ml

```diff
@@ -0,0 +1,79 @@
+module Identity = struct
```

Comment:
it might be more appropriate to put `tar_helpers.ml` in the util library (i.e. move it to the util folder)

Reply:
Okay